### DISCUSSION TITLE
error TS2307: Cannot find module './useCollection' or its corresponding type declarations.  10 export { useCollection } from './useCollection';

### DISCUSSION THREAD

**Discussion by CassioLuis** [unovue/reka-ui#508] [General]
![image_2023-11-09_16-21-10](https://github.com/radix-vue/radix-vue/assets/21344396/8113a6ed-8615-4cab-ba65-33ca127fb3bd)

this useCollection simply does not exist and is being exported, this is generating an error in 'npm run build'. Any help will be appreciated.

**Comment by zernonia** [comment]
Thanks @CassioLuis . I converted this into issue and will be handled swiftly!

**Reply by zernonia** [reply]
Fixed in https://github.com/radix-vue/radix-vue/pull/510

**Reply by CassioLuis** [reply]
thank you very much !
